include/mbedtls/check_config.h

```diff
@@ -674,6 +674,23 @@
 #error "MBEDTLS_SSL_DTLS_ANTI_REPLAY  defined, but not all prerequisites"
```

Comment:
I think that's a leftover comment from a merge based on a previous version of the unremove-noncrypto branch, which used to introduce that duplication. Newer versions of unremove-non-crypto fixed that, so this comment is no longer accurate indeed.

Reply:
Removed "* `MBEDTLS_PARAM_FAILED`: this has been duplicated. Do remove the version from crypto below, because the version from tls above is slightly improved." from the commit message.